packages/lighthouse-spa/src/app/app.component.spec.ts

```diff
@@ -6,14 +6,15 @@ import { AppComponent } from './app.component';
 describe('AppComponent', () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      imports: [
+    imports: [
         RouterTestingModule
-      ],
-      declarations: [
+    ],
+    declarations: [
         AppComponent
-      ],
-      schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
-    }).compileComponents();
+    ],
+    schemas: [CUSTOM_ELEMENTS_SCHEMA],
+    teardown: { destroyAfterEach: false }
+}).compileComponents();
```

Comment:
Can you fix the indentation here?

And also in a few other files.
- [property-card.component.spec.ts](packages/lighthouse-spa/src/app/dashboard/components/property-card/property-card.component.spec.ts)#L12
- [dashboard.service.spec.ts](packages/lighthouse-spa/src/app/dashboard/dashboard.service.spec.ts)#L28
- etc.
You may need to go through the files and run an auto formatter.

Reply:
thanks, I will fix it!